package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack",
-    "dev": "webpack serve"     
+    "dev": "webpack serve"
```

Comment:
- [ ] Please make sure your Webpack dev serve is setup to run in the browser. You can do that by adding `--open` flag infront of the `"dev": "webpack serve"` so it will look like `"dev": "webpack serve --open"`